docs/mkdocs.en.yml

```diff
@@ -6,17 +6,19 @@ theme:
 nav:
 
   - Welcome: index.md
-  - Architecture Overview: architecture.md
-  - Automated Deployment: deployment.md
-  - Workshop: workshop/background.md
-  - Molecule Unfolding: workshop/a-molecule-unfolding/molecule-unfolding.md
-  - Notebook Experiment: workshop/a-molecule-unfolding/notebook-experiment.md
-  - Batch Test: workshop/a-molecule-unfolding/batch-test.md
-  - Test Your Own Model: workshop/a-molecule-unfolding/test-your-own-model.md
   - Cost: cost.md
+  - Architecture Overview: architecture.md
   - Security: security.md
   - Supported regions: regions.md
   - AWS CloudFormation template: template.md
+  - Automated Deployment: deployment.md
+  - Workshop: 
+    - Overview: workshop/background.md
+    - Molecule Unfolding: 
+      - Background: workshop/a-molecule-unfolding/molecule-unfolding.md
+      - Notebook Experiment: workshop/a-molecule-unfolding/notebook-experiment.md
+      - Batch Test: workshop/a-molecule-unfolding/batch-test.md
```

Comment:
should be `batch evaluation` for consistence?

Reply:
This will be improved in origin/feature/doc-workshop-batch-eyom

Reply:
@amliuyong

Reply:
fixed